PubSub/sdk/src/main/java/org/lantern/pubsub/AutoStart.java

```diff
@@ -0,0 +1,19 @@
+package org.lantern.pubsub;
```

Comment:
Yeah I kind of want to ditch getlantern in general honestly and to move everything to just lantern -- it confuses people constantly!